src/EmailReplyParser/Parser/EmailParser.php

```diff
@@ -36,6 +36,8 @@ class EmailParser
         '/^(Den\s.+\sskrev\s.+:)$/m', // Den DATE skrev NAME <EMAIL>:
         '/^(Am\s.+\sum\s.+\sschrieb\s.+:)$/m', // Am DATE um TIME schrieb NAME:
         '/^(.+\s<.+>\sschrieb:)$/m', // NAME <EMAIL> schrieb:
+        '/(From.*:.+\s<.+>)/m', // "From: NAME <EMAIL>"  OR "From : NAME <EMAIL>" (With support whitespace before start)
```

Comment:
Hi thanks for your review.
I add point because isn't really whitespace

De=C2=A0: Prénom NOM <personne@yahoo.fr>

=C2=A0 => non-breaking space